docs/src/app/app-routes.jsx

```diff
@@ -123,7 +123,7 @@ const AppRoutes = (
       <Route path="text-field" component={TextFieldPage} />
       <Route path="time-picker" component={TimePickerPage} />
       <Route path="toggle" component={TogglePage} />
-      <Route path="toolbars" component={ToolbarsPage} />
+      <Route path="toolbar" component={ToolbarPage} />
```

Comment:
Huh. Yes, I guest that got lost in the shuffle when I split the Toolbar changes into two commits. I'll fix it up.